opensearch_dashboards.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "searchRelevance",
+  "version": "2.3.0.0",
```

Comment:
Main should be for the next major release, version 3.0, when we have something working, we should cut a 2.x branch and update the version to 2.4.0.0. Finally, when we release 2.4.0, we cut a 2.4 branch from 2.x. https://github.com/opensearch-project/.github/blob/main/RELEASING.md#opensearch-branching Can this be changed? Also, I think the id should be searchRelevanceDashboards to be more specific: https://github.com/opensearch-project/opensearch-plugins/blob/main/CONVENTIONS.md#opensearch-plugins

Reply:
So we should change `version` from `2.3.0.0` to `3.0` here?

Reply:
yes, please